src/v/pandaproxy/api/api-doc/schema_registry_definitions.def.json

```diff
@@ -91,6 +91,22 @@
         }
       }
     },
+    "get_iceberg_config": {
+      "type": "object",
+      "properties": {
+        "compatibilityMode": {
+          "type": "string"
+        }
+      }
+    },
+    "put_iceberg_config": {
+      "type": "object",
+      "properties": {
+        "compatibilityMode": {
+          "type": "bool"
```

Comment:
Should we use an enum here so we can have more nuanced (or more or less strict) checks in the future?

Reply:
yeah i have a todo for that. was thinking about a "warn" mode if you just wanted to alert on a log line but not disrupt producers